src/test/java/uk/gov/hmcts/reform/civil/bpmn/NotifyJudgmentVariedDeterminationOfMeansTest.java

```diff
@@ -46,14 +47,23 @@ void shouldSuccessfullyNotifyJudgmentVariedDeterminationOfMeans(boolean twoRepre
             START_BUSINESS_ACTIVITY,
             variables
         );
+        //complete call to CJES for edit Judgment
+        ExternalTask sendJudgmentDetailsToCJES = assertNextExternalTask(PROCESS_CASE_EVENT);
+        assertCompleteExternalTask(
+            sendJudgmentDetailsToCJES,
+            PROCESS_CASE_EVENT,
+            "SEND_JUDGMENT_DETAILS_CJES",
+            "SendJudgmentDetailsToCJES"
+        );
 
         //complete the notification to Claimant
         ExternalTask claimantNotification = assertNextExternalTask(PROCESS_CASE_EVENT);
         assertCompleteExternalTask(
             claimantNotification,
             PROCESS_CASE_EVENT,
             "NOTIFY_CLAIMANT_JUDGMENT_VARIED_DETERMINATION_OF_MEANS",
-            "NotifyClaimantJudgmentVariedDeterminationOfMeans"
+            "NotifyClaimantJudgmentVariedDeterminationOfMeans",
+            variables
         );
```

Comment:
i think another test can be added for when judgmentRecordedReason is not DETERMINATION_OF_MEANS

Reply:
will add test case for this